python/equistore-operations/equistore/operations/manipulate_dimension.py

```diff
@@ -312,8 +312,8 @@ def remove_dimension(tensor: TensorMap, axis: str, name: str) -> TensorMap:
     if axis == "keys":
         keys = keys.remove(name=name)
 
-    blocks = []
-    for block in tensor:
+    blocks: List[TensorBlock] = []
+    for _, block in tensor.items():
```

Comment:
```suggestion
    for block in tensor.blocks():
```

(just for style and consistency with other scriptable functions)

Reply:
done